arm-web/2016-09-01/swagger/AppServicePlans.json

```diff
@@ -2561,255 +2561,244 @@
     "SiteConfig": {
       "description": "Configuration of an App Service app.",
       "type": "object",
-      "allOf": [
-        {
-          "$ref": "#/definitions/Resource"
-        }
-      ],
```

Comment:
Confirming - Is this intentional? removal of "Resource" hence making SiteConfig not an ARM tracked resource anymore?